testsuite/diem-swarm/src/swarm.rs

```diff
@@ -203,15 +297,23 @@ impl AsRef<Path> for DiemSwarmDir {
     }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum NodeType {
+    Validator,
+    ValidatorFullNode,
+    PublicFullNode,
+}
+
 /// Struct holding instances and information of Diem Swarm
 pub struct DiemSwarm {
+    label: &'static str,
```

Comment:
What's the purpose of this?

Reply:
So, this is what became really hard to debug.  I had to use a swarm as a "node" or a set of "nodes".  Type isn't enough to determine the difference between multiple full node swarms.  This just allows us to do things like, print out what exact swarm you were using for things like the log printout on failure.  Before this I tried type, and it went like below, and in random orders (making it even more confusing).  I had to determine which logs were which by looking at the node config.
```
Validator logs: ...
ValidatorFullNode logs: ...
FullNode logs: ...
FullNode logs: ...
```